dll/shellext/zipfldr/CZipCreator.cpp

```diff
@@ -279,6 +292,7 @@ unsigned CZipCreatorImpl::JustDoIt()
 
     int err = 0;
     CStringW strTarget, strBaseName = DoGetBaseName(m_items[0]);
+    BOOL bUtf8 = IsUtf8ZipEnabled();
```

Comment:
Why would anyone need to disable this? Also the knowledge about this setting will be lost immediately

Reply:
I think runtime detection would be better, e.g. if one of filenames cannot be converted to ANSI losslessly, UTF-8 needs to be enabled.

Reply:
Configuration is deleted.